lib/filesystem.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const fs = require('fs');
+
+
+
```

Comment:
@loganlsr a few extra line breaks here - there should only be one line break between your readFile function and your fs require statement.